EIPS/eip-3333.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
+discussions-to: https://discord.gg/3sjBbWKM4E (ETH R&D 1559-mining)
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-11
+---
+
+## Simple Summary
+Changes the block reward paid to proof-of-work (POW) miners to 3 ETH from existing 2 ETH, but kicks off a 0.25 quarterly drop to a final 1.0 ETH Reward 2 Years Later.
+
+## Abstract
+Starting with `FORK_BLKNUM` block rewards will be increased to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly. Every 3 months after, measured by EPOCH count, Block Reward drops 0.25 every quarter till Ethereum Block Reward reaches 1.0 BR per Block.
+
+## Motivation
+With the passing of EIP-1559, the potential steep reduction of Ethereum Miner fee's, which currently are averaging a doubling of the exiting block reward have several scenerios where it can become the 50% key factor on keeping ethereum mining participation growing the current all time high hashpower, all the way to the London hard fork release. There are several scenerios where this can create a steep hashrate decrease when the BASEFEE starts to get burned vs distributed to miners. This can create an exteciential risk to ethereum's network by incentivizing miners to move to more profitable solutions, including broker services such as Nicehash and Miningrigrentals. To ward off this threat, a recommend action is to temporarly increase Eth Mining Rewards by +1 offset then began to decrease -2 Eth over 8 quarters, 2 full years from London's Hardfork. This should align well with the current optimisitic ETH 2.0 PoW/PoS handoff and appropriately set Eth Inflation on a managed reduction against a schedule that can be planned for. Additionally this provides a higher incentive for price discovery which can offset any further steep impacts for the mining security budget. 
+
+## Specification
+#### Adjust Block, Uncle, and Nephew rewards
+Adjust the block reward to `new_block_reward`, where
+
+    new_block_reward = 3_000_000_000_000_000_000 if block.number >= FORK_BLKNUM else block.reward
+
+(5E18 wei, or 3,000,000,000,000,000,000 wei, or 3 ETH).
+
+Analogue, if an uncle is included in a block for `block.number >= FORK_BLKNUM` such that `block.number - uncle.number = k`, the uncle reward is
+
+    new_uncle_reward = (8 - k) * new_block_reward / 8
+
+This is the existing formula for uncle rewards, simply adjusted with `new_block_reward`.
+
+The nephew reward for `block.number >= FORK_BLKNUM` is
+
+    new_nephew_reward = new_block_reward / 32
+
+This is the existing formula for nephew rewards, simply adjusted with `new_block_reward`.
+
```

Comment:
````suggestion
## Specification
### Constants
* `TRANSITION_START_BLOCK_NUMBER: TBD`
* `TRANSITION_DURATION: 8_000_000` (about two years)
* `TRANSITION_END_BLOCK_NUMBER: FORK_BLOCK_NUMBER + TRANSITION_DURATION`
* `STARTING_REWARD: 3_000_000_000_000_000_000`
* `ENDING_REWARD: 1_000_000_000_000_000_000`
* `REWARD_DELTA: STARTING_REWARD - ENDING_REWARD`

### Block Reward
```py
if block.number >= TRANSITION_END_BLOCK_NUMBER:
    block_reward = ENDING_REWARD
elif block.number >= TRANSITION_START_BLOCK_NUMBER:
    block_reward = STARTING_REWARD - REWARD_DELTA * TRANSITION_DURATION / (block.number - TRANSITION_START_BLOCK_NUMBER)
```
````
A smooth transition is generally preferable to cliffs, for the exact reasons outlined in the Motivation section.  😄 

Also, no need to mention uncle or nephew rewards since those formulas aren't changing.